src/org/sosy_lab/java_smt/solvers/cvc4/CVC4FloatingPointFormulaManager.java

```diff
@@ -109,7 +109,7 @@ protected Expr makeNumberAndRound(String pN, FloatingPointType pType, Expr pRoun
 
     final Rational rat = toRational(pN);
     final BigInteger upperBound =
-        getBiggestNumberBeforeInf(pType.getMantissaSize(), pType.getExponentSize());
+        getBiggestNumberBeforeInf(pType.getMantissaSizeWithSignBit(), pType.getExponentSize());
```

Comment:
This is a behavior change AFAIS, is this desired?

Reply:
No, good spot! Changed. Thank you!